keps/sig-node/2008-checkpoint-restore/README.md

```diff
@@ -0,0 +1,328 @@
+# KEP-2008: Add checkpoint and restore to the API
```

Comment:
The title should be specific to cover the current scope.

Reply:
Changed the title and only focused on the Forensic Container Checkpointing idea. I removed almost everything around container migration for now.